packages/neuron-wallet/src/listeners/main/index.ts

```diff
@@ -21,7 +21,7 @@ export const register = () => {
 
   MultisigConfigDbChangedSubject.getSubject()
     .pipe(debounceTime(500))
-    .subscribe(async (event: string) => {
+    .subscribe(async event => {
```

Comment:
Why is the type removed

Reply:
Because the type can be inference by https://github.com/nervosnetwork/neuron/pull/2632/files#diff-0eadefcf991daa7cae52f65fbea7dd80eabf9f4b5cd3f0392d3dc0ec42b348f5R5